### ISSUE TITLE
Option to let the scenario choose the investigators.

### ISSUE THREAD

**Issue by redwolf2** [NPBruce/valkyrie#488]
Instead of having the player choose the investigators it would be nice if you could add a fixed set of investigators in the scenario. This would allow for more character themed scenarios and might be handy functionality for this issues: https://github.com/NPBruce/valkyrie/issues/487 https://github.com/NPBruce/valkyrie/issues/290. Also sorry for all the feature request, I will keep it more focused on errors in the future.

**Comment by cbusbird** [comment]
I agree that streamlining the testing process in anyway would be an improvement.  I have events and tokens that behave differently depending on the number of investigators playing the game and if this gets implemented it would be nice to be able to define the number of investigators or maybe just have the ability to change it from a default of 2.

**Comment by NPBruce** [comment]
It's a big hack, but now that there are #hero vars you could tell the user what to select and error if they don't.
